test/test_cuda.py

```diff
@@ -4839,9 +4839,7 @@ def create_mempool_and_make_active():
         self.assertEqual(len(set(active_pool_ids)), 4)
 
     def test_mempool_expandable(self):
-        torch.cuda.memory._set_allocator_settings(
-            "expandable_segments:True"
-        )
+        torch.cuda.memory._set_allocator_settings("expandable_segments:True")
```

Comment:
Quick question: does this need to be reset? will this setting propagate to other tests that run after?

Reply:
Good point, I'll reset anyway.